src/riak_core_console.erl

```diff
@@ -752,6 +753,27 @@ commit_staged([]) ->
                       "plan and try to commit again~n")
     end.
 
+timezone([]) ->
```

Comment:
Ok. We're not returning anything from any of these (at least, we're not returning anything anyone cares about). riak-admin calls into this module with the expectation that the module will print results for the user to read.